src/views/blocks/attendees/view-link.php

```diff
@@ -29,6 +29,18 @@
 	die( '-1' );
 }
 
+/**
+ * Filters the arg to determine whether or not to hide the view link.
+ *
+ * @since TBD
+ *
+ * @param bool True/false to hide view link.
+ */
+$hide_view_link = apply_filters( 'tec_tickets_hide_view_link', tribe_doing_shortcode( 'tribe_tickets' ) );
```

Comment:
We are intentionally trying to avoid having filters in templates. Can we possibly move this one to where the template is invoked and send that as a parameter?

We'll need to document the parameter in the doblocks. And also document the `@since TBD` describing the changes. And set the `@version TBD`.